Docs/user_guide/model_guidelines.rst

```diff
@@ -1,28 +1,30 @@
 :orphan:
 
-============================
+############################
 Model Guidelines for PyTorch
-============================
+############################
 
-To implement the Cross Layer Equalization API,  aimet_torch.cross_layer_equalization.equalize_model(),  AIMET creates a computing graph to analyze the sequence of Operations in the model.
-If your model is defined using certain constructs, it restricts AIMET from successfully creating and analyzing the computing graph. The following table lists the potential issues and workarounds.
+To implement the Cross Layer Equalization API,  `aimet_torch.cross_layer_equalization.equalize_model()`,  AIMET creates a computing graph to analyze the sequence of operations in the model.
 
-Note: These restrictions are not applicable, if you are using the **Primitive APIs**
+Certain model constructs prevent AIMET from creating and analyzing the computing graph. The following table lists these potential issues and workarounds.
 
+.. admonition NOTE::
```

Comment:
This directive won't be rendered correctly, right? Either we should use specific admonition `.. note::` or generic admonition `..adomonition::`?  https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#admonitions-messages-and-warnings

Reply:
Changed to ".. note::"